ddtrace/contrib/aiohttp/patch.py

```diff
@@ -45,13 +47,25 @@ def __init__(self, obj, pin):
 
     async def connect(self, req, *args, **kwargs):
         pin = Pin.get_from(self)
-        with pin.tracer.trace("%s.connect" % self.__class__.__name__):
+        with pin.tracer.trace("%s.connect" % self.__class__.__name__) as span:
+            # set component tag equal to name of integration
+            span.set_tag("component", config.aiohttp.integration_name)
+
+            # set span.kind tag equal to type of request
+            span.set_tag(SPAN_KIND, SpanKind.SERVER)
+
```

Comment:
```suggestion
```

I am not sure if this should have span kind

Reply:
Agreed, removed any span kind tagging unless the span is a request span.